Remove VCF as viable format

JasperO98 opened the pull request:
Gene.iobio does not accept `vcf` as it has to find the `tbi` file which is only generated from the `vcf_bgzip` format.

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

bernt-matthias commented:
Could we generate the needed file on the fly in the tool?

JasperO98 commented:
Can we still retrieve a URL from that, if so then I can add it.

mvdbeek commented:
> Could we generate the needed file on the fly in the tool?

No, we can't change the input of a tool, which is what would be required for this to work. This would work if this was a display application, but I'm not sure that is better than the tool here.

mvdbeek reviewed (approved):
Looks good to me, but can you bump up the tool version on line 1 ? Not sure why this is version 4.7.1, but if that reflects something about iobio the new version should be `4.7.1+galaxy1`, otherwise you can pick any version higher than 4.7.1

JasperO98 commented:
> Looks good to me, but can you bump up the tool version on line 1 ? Not sure why this is version 4.7.1, but if that reflects something about iobio the new version should be `4.7.1+galaxy1`, otherwise you can pick any version higher than 4.7.1

I've corrected the version to `4.7.1+galaxy1`. 4.7.1 reflects the version of gene.iobio when we implemented this tool.

mvdbeek commented on tools/geneiobio/geneiobio-iframe.xml:
```diff
@@ -1,4 +1,4 @@
-<tool id="gene_iobio_display_generation_iframe" name="gene.iobio visualisation" version="4.7.1">
+<tool id="gene_iobio_display_generation_iframe" name="gene.iobio visualisation" version="4.7.1"+galaxy1>
```
```suggestion
<tool id="gene_iobio_display_generation_iframe" name="gene.iobio visualisation" version="4.7.1+galaxy1">
```
